src/session-storage-utils.ts

```diff
@@ -0,0 +1,5 @@
+export const getUserAuthenticated = (): string | null => sessionStorage.getItem('isAuthenticated');
+
+export const setUserAuthenticated = (isAuthenticated: boolean): void => {
+  sessionStorage.setItem('isAuthenticated', JSON.stringify(isAuthenticated));
+};
```

Comment:
this should be better a class, also add method `isAuthenticated`, which returns boolean athentication status.
`clear ` method is also needed, as we will need to clear session on logout